packages/curve-ui-kit/src/shared/ui/Balance.tsx

```diff
@@ -58,20 +59,22 @@ type BalanceTextProps<T> = {
 
 const BalanceText = <T extends Amount>({ symbol, balance, loading = false }: BalanceTextProps<T>) => (
   <WithSkeleton loading={loading}>
-    <Stack direction="row" gap={Spacing.xs} alignItems="center">
-      <Typography
-        className="balance"
-        variant="highlightS"
-        color={balance != null ? 'textPrimary' : 'textTertiary'}
-        data-testid="balance-value"
-      >
-        {balance == null ? '-' : formatNumber(balance, { abbreviate: true })}
-      </Typography>
+    <Tooltip title={t`Wallet balance`} body={`${balance} ${symbol}`} clickable>
```

Comment:
It seems that `balance` can be `null`
```suggestion
    <Tooltip title={t`Wallet balance`} body={notFalsy(balance, symbol).join(' ')} clickable>
```